.github/fabricbot.json

```diff
@@ -298,6 +298,51 @@
         "mergeType": "squash",
         "requireAllStatuses": true
       }
+    },
+    {
+      "taskType": "trigger",
+      "capabilityId": "IssueResponder",
+      "subCapability": "IssuesOnlyResponder",
+      "version": "1.0",
+      "config": {
+        "conditions": {
+          "operator": "and",
+          "operands": [
+            {
+              "name": "titleContains",
+              "parameters": {
+                "titlePattern": "typo/gi",
```

Comment:
What's the /gi for?

Reply:
supposed to represent global + case-insensitive